middleware/getResults.js

```diff
@@ -0,0 +1,41 @@
+const http = require('http');
+const async = require('async');
+const xmlParser = require('../lib/xmlParser');
+const utils = require('../lib/resultsUtils');
+const processResults = require('../lib/resultsProcessor.js');
+
+function getData(requestUrl, page, cb) {
```

Comment:
I was wondering exactly the same when I reviewed it myself. Should `cb` be the slightly longer `callback` or is it a known convention?

I'm happy with either and would probably prefer the latter to make it clearer. What do you think?